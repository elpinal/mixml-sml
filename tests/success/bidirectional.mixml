link
  link {type t = int} with
  link Y = {type u : Type} with
  {val f : int -> ext Y.u}
with
  link X = {module t = [type :Type]} with
  link {type u = bool} with
  {val f = fun (x : ext X.t) -> true}
