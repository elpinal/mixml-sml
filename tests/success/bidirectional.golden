{f : [int -> bool]+, t : [= int], u : [= bool]}
